src/main/scala/mesosphere/marathon/api/akkahttp/AkkaHttpModule.scala

```diff
@@ -66,7 +66,7 @@ class AkkaHttpModule(conf: MarathonConf with HttpConf) extends AbstractModule {
       pluginManager = pluginManager)
 
     val resourceController = new ResourceController
-    val systemController = new SystemController(config)
+    val systemController = new SystemController(conf, config, electionService)
```

Comment:
conf and config as two different parameters are very unclear. Can we find different (more specific name) for at least one of those parameters?

Reply:
+1

Reply:
I've renamed one to `marathonConfig`. It is a weird setup. Ideally we would just use the typesafe config.